Massive maintenance update.

pinchedloaf opened the pull request:
I have done the following:

- Integrated all existing pull requests from other forks of the current master for this repo.
- removed exact duplicates, keeping the files which were most descriptive of the game, version, etc.
- Integrated another CET dump, which brings the number of unique CT's up to approximately 10,000.
- Integrated additional CT's from inital repo creation.

- See additional pulls/commits for details on other changes:
- pruning bad CT's
- removing online game CT's
- some organization of games to directory.

----
Post acceptance, I'll be working on naming structures for versioning, attribution, labeling, etc. for CT's; which includes writing a bit of the specification in the core README.md file.

I'd be interested in maintainer access as well, if you are still looking for peeps to do it.

imm0r committed:
Create Grim Dawn - Crucible Mode

kaipozadia committed:
Update vanhelsing_x86_v1-4-1_steam_ce64_s10_aob_t10_154.ct

sunbeam906 committed:
Delete/Revise DarkSoulsIII.CT

Table contains online options that affect other people's simple game play. Either revise the table, removing some options or delete it altogether if too much hassle.

rcg4u committed:
added some of my own autoassembly scripts and some others I use quite a bit.

rcg4u committed:
added more tables i found

rcg4u committed:
uploaded new cheat engine tables, and added some tables I made for some games.  I made one for Star Trek Legacy.

rcg4u committed:
removed unreadable tables.

rcg4u committed:
removed a couple more bad tables, they say error no data when trying to open them

rcg4u committed:
uploaded latest of my tables

rcg4u committed:
modified the empires of the udnerworld script and removed sections not used at the moment.

rcg4u committed:
Added a rimworld script from fearlessrevolutions and universim script

rcg4u committed:
removed my old empires of the undergrowth script

rcg4u committed:
added a table for spider: rite of the shrouded moon

rcg4u committed:
added simcity 4 table and modified emergency 4 table and added army inf money

Commit:
Initial de-duplication effort.

Using fdupes, remove exact file duplicates, regardless of name.

Filtering:
- Preferred saving dupes that were pre-sorted by name.
- Kept highest 'version' number otherwise.

Commit:
Merge branch 'patch-1' of https://github.com/imm0r/CheatEngineTables

Merge non-mainline Grimm Dawn addition to repo.

Commit:
Merge branch 'patch-1' of https://github.com/kaipozadia/CheatEngineTables

Merge vanhelsing patch from kaipozadia. Will need to move this to x64 in
the filename, post merge.

Commit:
Cleaned up merge for van helsing

- Moved kaipozadia merge to x64 filename.
- Restored original x86 file.

Commit:
Merge branch 'master' of https://github.com/rcg4u/CheatEngineTables

Includes addition of new downloaded tables and some personal tables.

Commit:
Merge branch 'patch-1' of https://github.com/sunbeam906/CheatEngineTables

Deletion of DS3 online bullshittery.

Commit:
Lowercase all filenames for standardization

All files except README markdown have been lowercased.

Extensions:
find . -type d -execdir rename 's/(\.[A-Z]+$)/lc($1)/ge' *.*[A-Z]* \;

Files:
find . -type d -execdir rename 'y/A-Z/a-z/' * \;

With a bit of manual massaging.

Commit:
Add in additional CET archives and dedupe.

Using fdupes, remove exact file duplicates, regardless of name.

Filtering:
- Preferred saving dupes that were pre-sorted by name.
- Kept highest 'version' number otherwise.

Commit:
Cleanup of rcg4u commits to root directory.

Moved tables from root directory to personal tables directory from
rcg4u.

Commit:
Added additional cheat tables.

Commit:
Lowercase all filenames for standardization

All files except README markdown have been lowercased.

Extensions:
find . -type d -execdir rename 's/(\.[A-Z]+$)/lc($1)/ge' *.*[A-Z]* \;

Files:
find . -type d -execdir rename 'y/A-Z/a-z/' * \;

With a bit of manual massaging.

Commit:
Merge rcg4u's downloaded list into the tables location, deduping
any existing files.

Commit:
Moved public cheats into tables directory. Removed public cheats from rcg4u's
personal directory.

Commit:
Move rcg4u's tables into tables with rudimentary attribution; full
attribution once markdown format is finalized.

Commit:
Create LABELING.md

Initial draft for directory and table naming conventions for ct's.

Commit:
Update LABELING.md

Inline link table name format to respective sections.

Commit:
Added section for platform architecture.

Cleaned up markdown formatting.

Commit:
Add .verified nomenclature for finished game directories.

Begin sorting ct's by .exe into rought respective game directories.

exe's were found using:

grep -iroh "\(.*\.exe\)" . | tee ../tables-exe-search

with some manual post-processing.

Commit:
Initial sorting of ct's into directories.

Commit:
Initial sort of ct's into game directories.

Created directory for shinkangen's ct's for inital attribution.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sort of ct's into game directories.

Commit:
Initial sort of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Add specification for graphics API used in ct's.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game direcotories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game direcotries.

Commit:
Initital sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

w0 reviewed (commented):
Much needed update to directory format. However I believe Hexorg is not longer active based on his lack of recent commits. I'd consider focusing on your own repo. I'd glad help in it.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Added additional specifications for platform and DRM based on
observations while sorting.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Initial sorting of ct's into game directories.

Commit:
Added 2017-03-06 cet.archive.rar in its original form.

This is only added to have a full historical record of
cet.archive.rar. These files are already filtered in
the main tables location; and will be used to create a
tag before removing them.

Commit:
Removed original archive after being tagged. These are integrated
into the tables directory now.

Neyami committed:
Create FFXII_TZA.CT

From http://fearlessrevolution.com/viewtopic.php?f=4&t=5945

Neyami committed:
Add files via upload

From https://steamcommunity.com/linkfilter/?url=http://fearlessrevolution.com/download/file.php?id=8312

Commit:
Merge branch 'master' of https://github.com/Neyami/CheatEngineTables
